server/boards/app/blocks.go

```diff
@@ -408,25 +408,39 @@ func (a *App) DeleteBlockAndNotify(blockID string, modifiedBy string, disableNot
 		return nil
 	}
 
-	err = a.store.DeleteBlock(blockID, modifiedBy)
-	if err != nil {
-		return err
-	}
-
 	if block.Type == model.TypeImage {
 		fileName, fileIDExists := block.Fields["fileId"]
 		if fileName, fileIDIsString := fileName.(string); fileIDExists && fileIDIsString {
 			filePath := filepath.Join(block.BoardID, fileName)
```

Comment:
I realize this line existed previously, but I'm not sure this works. I believe "teamID" needs to be included as well. 
https://github.com/mattermost/mattermost-server/blob/master/server/boards/app/files.go#L104